hack/update-static.yaml.sh

```diff
@@ -18,6 +18,8 @@ cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/rbac.yaml
 cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/config.yaml $STATIC_DIR/03-trivy-operator.config.yaml
 cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/policies.yaml $STATIC_DIR/04-trivy-operator.policies.yaml
 cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/deployment.yaml $STATIC_DIR/05-trivy-operator.deployment.yaml
+cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/service.yaml $STATIC_DIR/05-trivy-operator.service.yaml
+cp $HELM_TMPDIR/trivy-operator-helm-template/trivy-operator/templates/servicemonitor.yaml $STATIC_DIR/05-trivy-operator.servicemonitor.yaml
 
 cat $CRD_DIR/vulnerabilityreports.crd.yaml \
```

Comment:
We need to add the service here as well, to aggregate it on `trivy-operator.yaml` (after adding it we need to run `./hack/update-static.sh`